binaries/scheduler/main.go

```diff
@@ -17,21 +19,22 @@ func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
 	flag.Parse()
 
-	parser := config.DefaultParser()
-	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
-	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
-	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	env, o := config.Default()
```

Comment:
Here env is different than {devel, staging, prod}. It's probably a bad name because of that collision.

The env here is then populated based on what we parse from the command-line (that, in Aurora, is passed from the config) in the line ParseAndEval.

The point of config.Default() is: multiple binaries can share much of the same environment. This is useful so that closed-source Scoot binaries can use much of the same from open source, but we haven't proven it out.

It gets to how much we value implicit (fewer lines of code; yay!) vs. explicit (easier to understand; yay!). I'm happy with other choices.